core/common/src/test/java/alluxio/util/executor/ControllableSchedulerTest.java

```diff
@@ -15,6 +15,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
+import org.junit.Assert;
```

Comment:
Instead of adding this, could you use the static imports in the `cancel` methods?

Reply:
done